gql_example_flutter/pubspec.yaml

```diff
@@ -0,0 +1,74 @@
+name: gql_example_flutter
+description: A new Flutter project.
```

Comment:
You can drop the `description` and run `dart ./multipack/bin/main.dart pubspec clean` before commit. It adds some stupid EOL whitespaces my editor tends to delete, but at least it guarantees consistency and reduces git diff noise.

Reply:
done in 719a806